frontend/src/components/modules/LinkObject.tsx

```diff
@@ -15,24 +15,38 @@ export const LinkObject = ({
   const isHome = url === "/";
   const target = isRelative ? undefined : "_blank";
   const rel = isRelative ? undefined : "noopener noreferrer";
+  const noLink = url === "##nolink";
```

Comment:
@Sun-Mountain I would rather this logic not exist on the `LinkObject` component. It starts to muddy the waters of what this component is for. On line `51` of the `NavMenu` component, there is similar logic for when a url is present. You use utilize that pattern on line `86` in the `NavMenu` component and should get the similar result. Let me know if that made sense